doc/rbd/nvmeof-initiator-linux.rst

```diff
@@ -0,0 +1,80 @@
+==============================
+ NVMe/TCP Initiator for Linux
+==============================
+
+Prerequisites
+=============
+
+-  Red Hat Enterprise Linux 9.2 or later
+
+Installation
+============
+
+1. Install the nvme-cli:
+
+   .. prompt:: bash #
+   
+      yum install nvme-cli
+
+2. Install the NVMe-oF module:
+
+   .. prompt:: bash # 
+   
+      modprobe nvme-fabrics
+
+3. Verify the NVMe/TCP target is reachable:
+
+   .. prompt::  bash #
+   
+      nvme discover -t tcp -a GATEWAY_IP -s 4420
+
+4. Connect to the NVMe/TCP target:
+
+   .. prompt:: bash #
+   
+      nvme connect -t tcp -a GATEWAY_IP -n SUBSYSTEM_NQN
+
+Next steps
+==========
+
+Verfiy that the initiator is set up correctly:
+
+1. List the NVMe block devices:
+
+   .. prompt:: bash #
+   
+      nvme list
+
+2. Create a filesystem on the desired device:
+
+   .. prompt:: bash #
+   
+      mkfs NVME_NODE_PATH
```

Comment:
My recollection is that bare `mkfs` creates ext2 filesystems by default.  I don't think we want to suggest that:
```suggestion
      mkfs.ext4 NVME_NODE_PATH
```

Reply:
Remedied.